packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -144,6 +144,16 @@ abstract class BindingBase {
           };
         }
       );
+      registerServiceExtension(
+          name: 'logging',
+          callback: (Map<String, Object> parameters) async {
+            final String eventKey = parameters['eventKey'];
```

Comment:
have the param be named `channel`?

Reply:
I like that.  I felt like `stream` wasn't quite right either...  I'll update elsewhere too.  Thanks!